cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -1423,15 +1423,14 @@ private long insertDocumentHistoryToNewTree(SQLDatabase db, DocumentRevision new
         callable.available = !newRevision.isDeleted();
         long newLeafSeq = callable.call(db);
 
-        pickWinnerOfConflicts(db, docNumericID, newRevision.getId(), previousLeafSeq);
+        pickWinnerOfConflicts(db, docNumericID, newRevision.getId());
```

Comment:
Do we need to keep the `previousLeafSeq` variable? 
https://github.com/cloudant/sync-android/pull/334/files#diff-3171a584301b2c5d64276721c439450aR1405